src/Model.js

```diff
@@ -447,12 +447,21 @@ class Model {
    * @method _makeObservable
    */
   _makeObservable (initialAttributes) {
-     const { defaultAttributes } = this
-
-     extendObservable(this, {
-       ...defaultAttributes,
-       ...initialAttributes
-     })
+     const { defaultAttributes, attributeNames } = this
+     // Make sure to also include id, relationships, and store
+     const names = [...attributeNames, 'id', 'relationships', 'store']
+
+     // NOTE: Discuss behavior or attribute defaults
```

Comment:
If the initial attribute is not present the default value will be used instead.

Reply:
This caused a test to fail and made me wonder if we should think about this behavior. In practice, right now we are not using the default values in many places, but I think it is still worth talking through.